adp-algorithms/mind/src/main/java/de/metanome/algorithms/mind/MindAlgorithm.java

```diff
@@ -57,17 +52,10 @@ public void setTableInputConfigurationValue(final String identifier,
       final TableInputGenerator... values) {
 
     if (identifier.equals(ConfigurationKey.TABLE.name())) {
-      builder.tableInputGenerators(asList(values));
-    }
-  }
-
-  @Override
-  public void setDatabaseConnectionGeneratorConfigurationValue(final String identifier,
-      final DatabaseConnectionGenerator... values) {
-    if (identifier.equals(ConfigurationKey.DATABASE.name())) {
-      builder.databaseConnectionGenerator(values[0]);
+      DatabaseConnectionGenerator[] databaseConnectionGenerators = { values[0].getDatabaseConnectionGenerator() };
```

Comment:
I would maybe not restrict the DatabaseConnectionGenerators to values[0]. Isn't it possible to specify multiple tables in metanome? Therefore maybe all of them are needed.

Reply:
`de.metanome.validation.ValidationConfigurationRequirements#acceptDatabaseConnectionGenerator` will use only the first one, too. In theory, yes, but we currently cannot deal with queries that span multiple databases or even multiple database servers. 
The API of `ValidationConfigurationRequirements` should reflect that. The signature was a good fit for forwarding from `setDatabaseConnectionGeneratorConfigurationValue `, now it's just clunky.